db/migrate/20190521175543_add_new_collection_location.rb

```diff
@@ -0,0 +1,19 @@
+class AddNewCollectionLocation < ActiveRecord::Migration[5.1]
+  # Data migration to add new intelligent collection location MetadataField
+
+  def up
+    existing = MetadataField.find_by(name: "new_collection_location")
+    unless existing
+      field = MetadataField.create(name: "new_collection_location", display_name: "New Collection Location", description: "Location sample was originally collected", base_type: 3, group: "Sample", host_genomes: HostGenome.all)
```

Comment:
What is the migraiton plan for the transition from collection_location to new_collection_location?

Reply:
added "Data Migration Plan" here https://czi.quip.com/gzWdAafIX8Yb/LocationMapping-Sprint-Plan